src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/LocalizedFormMetadataCollection.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Metadata\FormMetadata;
+
+/**
+ * Collection to store the localized FormMetadata objects in each locale.
+ */
+class LocalizedFormMetadataCollection
+{
+    /**
+     * @var FormMetadata[]
+     */
+    private $items = [];
+
+    public function add($locale, $formMetadata)
+    {
+        $this->items[$locale] = $formMetadata;
+    }
+
+    public function get($locale): FormMetadata
+    {
+        if (!isset($this->items[$locale])) {
+            throw new \InvalidArgumentException(sprintf('Locale "%s" does not exist in collection.', $locale));
+        }
+
+        return $this->items[$locale];
+    }
+
+    public function getItems(): array
```

Comment:
Here a phpdoc comment like the following would add a value, because there are no generics in PHP:

```php
/**
 * @return FormMetadata[]
 */
```